### PR TITLE
Fixed the problem of Init_repo.sh not working.

### PR DESCRIPTION
**Pull request by TomoyaKitaura**
The name of the variable was changed.

### CODE REVIEW

**Comment by zuiurs** [comment]
Thanks for your contribution.

This is like a bug due to format of the git source URL.
At the moment, `./tools/env` only supported a ssh format...

Could you modify `./tools/env` to correnspond "ssh" and "https" format?

**Comment by TomoyaKitaura** [comment]
Thank you for the confirmation.
My problem is solved.

And I made it support HTTP format.

File: tools/env [hunk 1]
```diff
@@ -4,7 +4,16 @@ set -eu
 export CLUSTER_NAME=${CLUSTER_NAME:-testbed}
 export LOADBALANCER_IP_NAME=${LOADBALANCER_IP_NAME:-testbedip}
 export KUBERNETES_PLATFORM=${KUBERNETES_PLATFORM:-gke}
-export TB_GITHUB_ORG_NAME=$(git config --get remote.origin.url | cut -d ":" -f 2 | cut -d "/" -f 1)
+
+GET_REMOTE_ORIGIN_URL=$(git config --get remote.origin.url)
+
+if [ -n "$(echo ${GET_REMOTE_ORIGIN_URL} | grep -E ^https://github.com)" ]; then
+  export TB_GITHUB_ORG_NAME=$(git config --get remote.origin.url | cut -d "/" -f 4)
```
Comment (zuiurs) [inline tools/env]: this can be `TB_GITHUB_ORG_NAME=$(echo ${REMOTE_ORIGIN_URL} | ...)`

File: tools/env [hunk 2]
```diff
@@ -4,7 +4,16 @@ set -eu
 export CLUSTER_NAME=${CLUSTER_NAME:-testbed}
 export LOADBALANCER_IP_NAME=${LOADBALANCER_IP_NAME:-testbedip}
 export KUBERNETES_PLATFORM=${KUBERNETES_PLATFORM:-gke}
-export TB_GITHUB_ORG_NAME=$(git config --get remote.origin.url | cut -d ":" -f 2 | cut -d "/" -f 1)
+
+GET_REMOTE_ORIGIN_URL=$(git config --get remote.origin.url)
```
Comment (zuiurs) [inline tools/env]: This variable name may be preferably `REMOTE_ORIGIN_URL`.

File: tools/init_repo.sh [hunk 3]
```diff
@@ -10,11 +10,11 @@ git checkout develop
 for DIR in manifests microservices; do
   for FILE in `find ${CURRENT_DIR}/../${DIR}/ -type f`; do
     perl -pi -e "s|__LOADBALANCER_IP_ADDRESS__|${LOADBALANCER_IP_ADDRESS}|g" $FILE;
-    perl -pi -e "s|__TB_GITHUB_ORG_NAME__|${TB_GITHUB_ORG_NAME}|g" $FILE;
```
Comment (zuiurs) [inline tools/init_repo.sh]: It seems `${TB_GITHUB_ORG_NAME}` is correct here 🙏

**Review by zuiurs — CHANGES_REQUESTED** [review]
Thank you for your fix.
Please check my reviews out!

File: tools/init_repo.sh (hunk 3 above)
Comment (TomoyaKitaura) [reply tools/init_repo.sh]: sorry. I fixed it.
I had made an innocent mistake.:pray::pray:

File: tools/env (hunk 2 above)
Comment (TomoyaKitaura) [reply tools/env]: I fixed it.

File: tools/env (hunk 1 above)
Comment (TomoyaKitaura) [reply tools/env]: I fixed it.
Is this correct?

**Review by zuiurs — APPROVED** [review]
LGTM

Thanks!

**Pull request merged**